indexing-service/src/test/java/io/druid/indexing/overlord/TestRemoteTaskRunnerConfig.java

```diff
@@ -33,7 +33,8 @@ public Period getTaskAssignmentTimeout()
   @Override
   public long getMaxZnodeBytes()
   {
-    return 1000;
+    // make sure this is large enough, otherwise RemoteTaskRunnerTest might fail unexpectedly
```

Comment:
Ok, and the exception that was thrown clearly pointed to updating this value, is that correct?